tests/Fixtures/Entity/ResetPasswordRequestTestFixture.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the SymfonyCasts ResetPasswordBundle package.
+ * Copyright (c) SymfonyCasts <https://symfonycasts.com/>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SymfonyCasts\Bundle\ResetPassword\Tests\Fixtures\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use SymfonyCasts\Bundle\ResetPassword\Model\ResetPasswordRequestInterface;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ * @author Ryan Weaver   <ryan@symfonycasts.com>
+ *
+ * @internal
+ * @ORM\Entity(repositoryClass="SymfonyCasts\Bundle\ResetPassword\Tests\Fixtures\ResetPasswordRequestRepositoryTestFixture")
+ */
+final class ResetPasswordRequestTestFixture implements ResetPasswordRequestInterface
```

Comment:
I'm on the fence as to whether use as is, or replace methods with `use ResetPasswordRequestTrait`. Thoughts?

Reply:
Um... I can see it both ways. I guess using the trait would just help cut down on code since I don't have a strong preference. The trait should be tested elsewhere, so in theory, it should be safe to use it here.

Reply:
I started to switch this to the trait and the test blew up (scope related). I'm purposely using public properties in this fixture to get away from a bloated test with getters/setters etc.

The trait is covered in our unit tests and I think i'm going to be writing a high level "everything works as intended" integration test. That would cover the trait on both front's. Plus, while not exactly reliable from the bundle's perspective, we will have the maker tests as well...